src/containers/Navbar2/Navbar.container.tsx

```diff
@@ -0,0 +1,48 @@
+import { connect } from 'react-redux'
```

Comment:
The directory and the component is called `Navbar2`, what do you think about renaming these files to `Navbar2`?